installer.sh

```diff
@@ -405,20 +405,37 @@ case ${ANSWER:0:1} in
   ;;
 esac
 
-# aha for html generation - future extension of emba
-#echo -e "\\n""$ORANGE""$BOLD""Downloading aha""$NC"
-#if ! [[ -f "external/aha" ]] ; then
-#  apt-get install make
-#  wget https://github.com/theZiz/aha/archive/master.zip -O external/aha-master.zip
-#  unzip ./external/aha-master.zip -d ./external
-#  rm external/aha-master.zip
-#  cd ./external/aha-master || exit 1
-#  echo -e "$ORANGE""$BOLD""Compile aha""$NC"
-#  make
-#  cd ../.. || exit 1
-#  mv "external/aha-master/aha" "external/aha"
-#  rm -R external/aha-master
-#else
-#  echo -e "$ORANGE""aha is already downloaded and compiled""$NC"
-#fi
+
+# aha for html generation
+
+echo -e "\\nTo use the emba report generator, we need a html file generator. aha will be installed via pip3."
+
+echo -e "\\n""$MAGENTA""$BOLD""Do you want to download and install aha (if not already on the system)?""$NC"
+read -p "(y/N)" -r ANSWER
```

Comment:
There are some changes in the master branch:

```
...
if [[ "$FORCE" -eq 0 ]] ; then
  echo -e "\\n""$MAGENTA""$BOLD""Do you want to download these applications (if not already on the system)?""$NC"
  read -p "(y/N)" -r ANSWER
else
  echo -e "\\n""$MAGENTA""$BOLD""These applications (if not already on the system) will be downloaded!""$NC"
  ANSWER=("y")
fi
case ${ANSWER:0:1} in
  y|Y )
...
```

Please apply these changes to your code.